### PR TITLE
[21]contact form validation

### PR DESCRIPTION
**Pull request by Iyiola-stack**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/peterIyiola/DOM/contact.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+let submitForm = document.getElementById('validateForm'),
```
Comment (adaugochi) [inline submissions/peterIyiola/DOM/contact.js:1]: Where are you using the js file in your HTML?

Comment (adaugochi) [reply submissions/peterIyiola/DOM/contact.js:1]: Send a screenshot of your result?

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/peterIyiola/DOM/contact.js (hunk 1 above)
Comment (Iyiola-stack) [reply submissions/peterIyiola/DOM/contact.js:1]: image
image

he js file(contact.js) is attached just before the closing body tag in my HTML

**Comment by Iyiola-stack** [comment]
Attended to

**Comment by Iyiola-stack** [comment]
Still awaiting your feedback

**Review by adaugochi — APPROVED** [review]

**Pull request merged**